yarn.lock

```diff
@@ -2,3 +2,7 @@
 # yarn lockfile v1
 
 
+"@types/react-select-country-list@^2.2.1":
```

Comment:
should this be added to `frontend/yarn.lock` ?

Reply:
i think you're right! updating now